pilot/cmd/pilot-agent/metrics/metrics.go

```diff
@@ -12,25 +12,37 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package status
+package metrics
 
-import "istio.io/pkg/monitoring"
+import (
+	"time"
+
+	"istio.io/pkg/monitoring"
+)
 
 var (
 	typeTag = monitoring.MustCreateLabel("type")
 
+	// StartupTime measures the time it takes for the agent to get ready Note: This
+	// is dependant on readiness probes. This means our granularity is correlated to
+	// the probing interval.
+	startupTime = monitoring.NewGauge(
```

Comment:
is this the kind of thing we care about aggregating across a group of proxies?  or is this just to look at a few individual results in perf monitoring ?

Reply:
I think individual ones can be useful to see outliers?

Reply:
Should we use "initialization_time_ms" stat of Envoy https://www.envoyproxy.io/docs/envoy/latest/configuration/observability/statistics.html?highlight=server%20uptime#server?

Not against adding a new stat but proxy already measures it - so just thinking if it is worth reusing it?

Reply:
I am not sure they are measuring the same thing. let me compare them